ansible/roles/ocp4-workload-debugging-workshop/defaults/main.yml

```diff
@@ -2,11 +2,15 @@
 become_override: False
 ocp_username: opentlc-mgr
 silent: False
+subdomain_base: "test.com"
+aws_region: "eu-west-1"
 num_users: 10
 
-_workshop_operator_namespace: workshop-infra
+workshop_openshift_user_name: userXX
+workshop_openshift_user_password: 'user_password'
+workshop_openshift_admin_password: 'admin_password'
 
-_workshop_user_password: openshift
+_workshop_operator_namespace: workshop-infra
```

Comment:
Ansible variables should not start with underscore, particularly not a variable that is in role defaults.

For Ansible, vars defined in defaults should be considered part of the role interface, variables meant to be overridden. A leading underscore traditionally marks a variable as private or internal. Internal role vars should be in `vars/main.yml` or defined at a task level.